docs/releases/v13/version-v13.0.0.md

```diff
@@ -159,7 +162,7 @@ We have introduced new metadata fields that will allow you to capture more usefu
 The v13 release features an upgraded PDF previewer, now powered by [PDF.js v4](https://github.com/mozilla/pdf.js). This update resolves previous issues with failed previews for certain PDF files, providing a more reliable and seamless viewing experience.
 
 ### Requests sharing
```

Comment:
```suggestion
We have introduced a new search filter in your dashboard so you can easily find records shared with you.
```

Reply:
> I don't know why it's saying I want to remove the heading 😅 the important thing is we shouldn't mention "review request" as we aren't introducing that in this release

It's the location of the code suggestion: if it was done on L165 it would target the sentence we want :grin: .